app/Http/Controllers/JobController.php

```diff
@@ -2,9 +2,97 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\Job;
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
 
 class JobController extends Controller
 {
-    //
+    /**
+     * Show job creation form.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function create()
+    {
+        return view('jobs.create');
+    }
+
+    /**
+     * Store job.
+     *
+     * @return \Illuminate\Contracts\Support\Renderable
+     */
+    public function store(Request $request)
```

Comment:
have you considered using special Request classes for every entity? from which you can handle authorisation and validation. at least maybe the main ones...